index.js

```diff
@@ -57,6 +57,22 @@ bot.registerCommand(
     }
 );
 
+bot.registerCommand(
+    "roll",
+    (msg) => {
+        try {
+            rollDecider(stripContent(msg.content));
+        } catch (err) {
+            return err
```

Comment:
Although using errors as strings is legal and convenient, as you've seen here, it means you lose a lot of potential debugging information. It's generally better to `throw new Error(...)`, and then this would be `return err.toString()`. Doing that includes backtrace support, if you ever need to debug this. It would also better handle any unexpected errors you might get.